pkg/messaging/nats/setup_test.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package nats_test
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+	"testing"
+
+	"github.com/mainflux/mainflux/logger"
+	"github.com/mainflux/mainflux/pkg/messaging"
+	"github.com/mainflux/mainflux/pkg/messaging/nats"
+	dockertest "github.com/ory/dockertest/v3"
+)
+
+var (
+	publisher messaging.Publisher
+	pubsub    messaging.PubSub
+)
+
+func TestMain(m *testing.M) {
+	pool, err := dockertest.NewPool("")
+	if err != nil {
+		log.Fatalf("Could not connect to docker: %s", err)
+	}
+
+	container, err := pool.Run("nats", "1.3.0", []string{})
+	if err != nil {
+		log.Fatalf("Could not start container: %s", err)
+	}
+	log.Println("NATS container created")
```

Comment:
Do we need this log for a test?

Reply:
When I do local tests, it's convinient. However, for automatic tests, I'm not sure... Can be removed.